api-scanner/src/test/java/net/corda/plugins/KotlinAnnotationsTest.java

```diff
@@ -95,4 +97,10 @@ public void testJvmDefaultAnnotation() throws IOException {
                  "##"
             );
     }
+
+    @Test
+    public void testFilteredMethodsAreExcluded() throws IOException {
+        assertThat(testProject.getApiLines())
+                .doesNotContain("  public <init>(int, String, int, kotlin.jvm.internal.DefaultConstructorMarker)");
```

Comment:
You can use something like `doesNotContainSequence("...")` to make this less brittle. I am concerned that we are sensitive to things like indentation and whitespace here.

Reply:
*Sigh*, _of course_ "sequence" doesn't mean "substring" in this context :man_facepalming:!